### DISCUSSION TITLE
how to install on win 10

### DISCUSSION THREAD

**Discussion by zambot8** [DISTRHO/Cardinal#122] [General]
Hi everyone, I'm a musician and I would like to try Cardinal vst on my pc (win 10) but I don't understand how to do it, which files to download . Please help me

**Comment by falkTX** [comment]
Use the "nightly" builds, seen from this link https://github.com/DISTRHO/Cardinal/actions/workflows/build.yml

Click on the latest successful build, then on the next page scroll down (note the canvas area is not scrollable with mouse wheel, weird github behaviour) and find the win64 build at the bottom.
After download and extraction, just move the CardinalFX.vst and CardinalSynth.vst to a place where your DAW/Host can find them.

**Reply by zambot8** [reply]
thx a lot!
